webapp/src/components/modal/subComponents/modalSubtitleAndError/styles.scss

```diff
@@ -10,3 +10,11 @@
     text-align: center;
   }
 }
+
+.modal__subtitle{
+  margin: 12px 0 0 0;
+}
+
+.modal__error{
+  margin: 12px 0 0 0;
```

Comment:
can't we simply use `margin-top` if the rest of the values are 0 or are we overriding other values?

Reply:
Yes, we are overriding the margin-bottom here